CONTRIBUTING.md

```diff
@@ -17,12 +17,14 @@ smooth collaboration, please follow the guidelines below.
 
 ## How to Contribute
 
-1. Fork the repository and create your branch from the `main` branch.
-2. Make your changes and test them thoroughly.
-3. Ensure your code follows our coding guidelines (see [Coding Guidelines](#coding-guidelines)).
-4. Commit your changes and provide a descriptive commit message.
-5. Push your changes to your forked repository.
-6. Submit a pull request to the `main` branch of our repository.
+1. Connect to [Gitpod](https://gitpod.io). If you don't have an account, create one with the Github provider (it's free!)
```

Comment:
Maybe let's add another section "How to Contribute using Gitpod"? This way the user can choose what they prefer